drivers/include/sen5x.h

```diff
@@ -0,0 +1,302 @@
+/*
+ * Copyright (C) 2023 TU Braunschweig Institut für Betriebssysteme und Rechnerverbund
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+ #ifndef SEN5X_H
+ #define SEN5X_H
+
+/**
+ * @defgroup    drivers_sen5x Sensirion Embedded I2C SEN5x Driver
+ * @ingroup     drivers_sensors
+ * @ingroup     drivers_saul
+ * @brief       Driver for I2C communication to SEN5x devices.
+ *
+ * @{
+ *
+ * @file
+ *
+ * @author      Daniel Prigoshij <prigoshi@ibr.cs.tu-bs.de>
+ */
+
+#include "periph/i2c.h"
+#include <stdint.h>
+#include "saul.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Wrapper for measured values
+ */
+typedef struct {
+    uint16_t mass_concentration_pm1p0;      /**< raw value is scaled with factor 10: PM1.0 [µg/m³] = value / 10 */
+    uint16_t mass_concentration_pm2p5;      /**< raw value is scaled with factor 10: PM2.5 [µg/m³] = value / 10 */
+    uint16_t mass_concentration_pm4p0;      /**< raw value is scaled with factor 10: PM4.0 [µg/m³] = value / 10 */
+    uint16_t mass_concentration_pm10p0;     /**< raw value is scaled with factor 10: PM10.0 [µg/m³] = value / 10 */
+    uint16_t number_concentration_pm0p5;    /**< raw value is scaled with factor 10: PM0.5 [#/cm³] = value / 10 */
+    uint16_t number_concentration_pm1p0;    /**< raw value is scaled with factor 10: PM1.0 [#/cm³] = value / 10 */
+    uint16_t number_concentration_pm2p5;    /**< raw value is scaled with factor 10: PM2.5 [#/cm³] = value / 10 */
+    uint16_t number_concentration_pm4p0;    /**< raw value is scaled with factor 10: PM4.0 [#/cm³] = value / 10 */
+    uint16_t number_concentration_pm10p0;   /**< raw value is scaled with factor 10: PM10.0 [#/cm³] = value / 10 */
+    uint16_t typical_particle_size;         /**< raw value is scaled with factor 1000: Size [µm] = value / 1000*/
+    int16_t ambient_humidity;               /**< raw value is scaled with factor 100: RH [%] = value / 100 */
+    int16_t ambient_temperature;            /**< raw value is scaled with factor 200: T [°C] = value / 200 */
+    int16_t voc_index;                      /**< raw value is scaled with factor 10: VOC Index = value / 10 */
+    int16_t nox_index;                      /**< raw value is scaled with factor 10: NOx Index = value / 10 */
+} sen5x_measurement_t;
+
+/**
+ * @brief   Device initialization parameters
+ */
+typedef struct {
+    i2c_t i2c_dev;                  /**< I2C device which is used */
+    uint8_t i2c_addr;               /**< I2C address */
+} sen5x_params_t;
+
+/**
+ * @brief   Device descriptor for the driver
+ */
+typedef struct {
+    sen5x_params_t params;                  /**< Device initialization parameters */
+} sen5x_t;
+
+/**
+ * @brief   Initialize the given device
+ *
+ * @param[inout] dev        Device descriptor of the driver
+ * @param[in]    params     Initialization parameters
+ * 
```

Comment:
```suggestion
 *
```

There are some static-tests errors about trailing whitespaces, you can check the files page of GitHub ( https://github.com/RIOT-OS/RIOT/pull/19955/files ) or run `make static-test` to run the static tests locally.